### PR TITLE
Update Telegraf Collection Architecture documentation

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update Telegraf Collection Architecture documentation, Changes related to Helm Chart v3

Issue number: https://jira.kumoroku.com/jira/browse/SUMO-206934

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by swiatekm-sumo — COMMENTED** [review]
Some minor phrasing adjustments.

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md [hunk 1]
```diff
@@ -38,46 +38,57 @@ You select an existing HTTP Source on a Hosted Collection as the destination for
...
+The diagram below illustrates where Telegraf fits into a Kubernetes environment monitored by Sumo Logic.
+In this example, we’re monitoring an Nginx in a Kubernetes cluster.
 
 ![Telegraf-K8s.png](/img/send-data/Telegraf-K8s.png)
 
-The [Telegraf Operator](https://github.com/influxdata/telegraf-operator), part of Node 1, monitors the dynamic nature of a Kubernetes cluster: pods coming and going, IP addresses changing. It defines a common output destination for all collected metrics and configures Sidecar monitoring on application pods using annotations. Note that pods have the annotations, and injects a Telegraf container to each. These pod annotations provide the instructions for the operator to configure each Telegraf sidecar.
+[Telegraf Operator](https://github.com/influxdata/telegraf-operator) intercepts requests related to pods in the cluster, using the mutating webhooks functionality in Kubernetes.
+Telegraf Operator reads the pod annotations in the request and if an annotation says to add a Telegraf sidecar, then Telegraf Operator adds that instance as an additional container within that pod.
+
+Metrics collection using Telegraf Operator is made in following process:
```
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md:49]: It doesn't summarised :P

[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md:49]: @kimsauce Could you help here?

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md [hunk 2]
```diff
@@ -38,46 +38,57 @@ You select an existing HTTP Source on a Hosted Collection as the destination for
...
 
-To summarize:
 1. User add annotations to the pods where they want Telegraf to collect data from.
 1. Prometheus connects to Telegraf via the user's annotations. Sumo Logic preconfigures Prometheus to fetch the metrics.
-1. Prometheus then sends metrics to Fluentd.
-1. Fluentd sends the metrics on to the HTTP Source running on a Sumo Logic Hosted Collector.
+1. Prometheus then sends metrics to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
+1. [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) sends the metrics on to Sumo Logic.
```
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md]: changed

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md [hunk 3]
```diff
@@ -38,46 +38,57 @@ You select an existing HTTP Source on a Hosted Collection as the destination for
...
 ![Telegraf-K8s.png](/img/send-data/Telegraf-K8s.png)
 
-The [Telegraf Operator](https://github.com/influxdata/telegraf-operator), part of Node 1, monitors the dynamic nature of a Kubernetes cluster: pods coming and going, IP addresses changing. It defines a common output destination for all collected metrics and configures Sidecar monitoring on application pods using annotations. Note that pods have the annotations, and injects a Telegraf container to each. These pod annotations provide the instructions for the operator to configure each Telegraf sidecar.
+[Telegraf Operator](https://github.com/influxdata/telegraf-operator) intercepts requests related to pods in the cluster, using the mutating webhooks functionality in Kubernetes.
+Telegraf Operator reads the pod annotations in the request and if an annotation says to add a Telegraf sidecar, then Telegraf Operator adds that instance as an additional container within that pod.
+
+Metrics collection using Telegraf Operator is made in following process:
 
-To summarize:
 1. User add annotations to the pods where they want Telegraf to collect data from.
```
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md]: changed

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md:49]: @kkujawa-sumo @swiatekm-sumo  I think it is summarizing the information above it. How about both?

_To summarize, here's how to collect metrics using the Telegraf Operator:_

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor suggestion

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md]: change to the form proposed by @kimsauce

**Comment by kkujawa-sumo** [comment]
@kimsauce please take another look

File: docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md [hunk 4]
```diff
@@ -38,46 +38,57 @@ You select an existing HTTP Source on a Hosted Collection as the destination for
...
-To summarize:
-1. User add annotations to the pods where they want Telegraf to collect data from.
+To summarize, here's how to collect metrics using the Telegraf Operator:
+
+1. User adds annotations to the pods where they want Telegraf to collect data from.
 1. Prometheus connects to Telegraf via the user's annotations. Sumo Logic preconfigures Prometheus to fetch the metrics.
-1. Prometheus then sends metrics to Fluentd.
-1. Fluentd sends the metrics on to the HTTP Source running on a Sumo Logic Hosted Collector.
+1. Prometheus then sends metrics to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
+2. [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) processes metrics, enriches metadata, and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md:54]:
```suggestion
2. The [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) processes metrics, enriches metadata, and sends metrics to Sumo Logic.
```

**Review by kimsauce — APPROVED** [review]
